### PR TITLE
Requiring Import (PR 8 of ...)

### PR DESCRIPTION
**Pull request by katbyte**
resources `S` to `SQL`

### CODE REVIEW

**Comment by katbyte** [comment]
```
[19:16:44] kt@snowbook:~/hashi/..3../terraform-providers/terraform-provider-azurerm▸refactor/require_import/S-SQL$ testazure TestAccAzureRMSc.*_requiresImport
==> Fixing source code with gofmt...
# This logic should match the search logic in scripts/gofmtcheck.sh
gofmt -s -w `find . -name '*.go' | grep -v vendor`
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./azurerm -v -test.run=TestAccAzureRMSc.*_requiresImport -timeout 180m -ldflags="-X=github.com/terraform-providers/terraform-provider-azurerm/version.ProviderVersion=acc"
=== RUN   TestAccAzureRMSchedulerJobCollection_requiresImport
=== PAUSE TestAccAzureRMSchedulerJobCollection_requiresImport
=== RUN   TestAccAzureRMSchedulerJob_web_requiresImport
=== PAUSE TestAccAzureRMSchedulerJob_web_requiresImport
=== CONT  TestAccAzureRMSchedulerJobCollection_requiresImport
=== CONT  TestAccAzureRMSchedulerJob_web_requiresImport
--- PASS: TestAccAzureRMSchedulerJobCollection_requiresImport (90.87s)
--- PASS: TestAccAzureRMSchedulerJob_web_requiresImport (95.82s)
PASS
ok  	github.com/terraform-providers/terraform-provider-azurerm/azurerm	96.280s
[11:32:12] kt@snowbook:~/hashi/..3../terraform-providers/terraform-provider-azurerm▸refactor/require_import/S-SQL$
```

**Comment by katbyte** [comment]
```
[19:31:23] kt@snowbook:~/hashi/..3../terraform-providers/terraform-provider-azurerm▸refactor/require_import/S-SQL$ testazure TestAccAzureRMSer.*_requiresImport
==> Fixing source code with gofmt...
# This logic should match the search logic in scripts/gofmtcheck.sh
gofmt -s -w `find . -name '*.go' | grep -v vendor`
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./azurerm -v -test.run=TestAccAzureRMSer.*_requiresImport -timeout 180m -ldflags="-X=github.com/terraform-providers/terraform-provider-azurerm/version.ProviderVersion=acc"
=== RUN   TestAccAzureRMServiceFabricCluster_requiresImport
=== PAUSE TestAccAzureRMServiceFabricCluster_requiresImport
=== RUN   TestAccAzureRMServiceBusNamespaceAuthorizationRule_requiresImport
=== PAUSE TestAccAzureRMServiceBusNamespaceAuthorizationRule_requiresImport
=== RUN   TestAccAzureRMServiceBusNamespace_requiresImport
=== PAUSE TestAccAzureRMServiceBusNamespace_requiresImport
=== RUN   TestAccAzureRMServiceBusQueueAuthorizationRule_requiresImport
=== PAUSE TestAccAzureRMServiceBusQueueAuthorizationRule_requiresImport
=== RUN   TestAccAzureRMServiceBusQueue_requiresImport
=== PAUSE TestAccAzureRMServiceBusQueue_requiresImport
=== RUN   TestAccAzureRMServiceBusSubscriptionRule_requiresImport
=== PAUSE TestAccAzureRMServiceBusSubscriptionRule_requiresImport
=== RUN   TestAccAzureRMServiceBusSubscription_requiresImport
=== PAUSE TestAccAzureRMServiceBusSubscription_requiresImport
=== RUN   TestAccAzureRMServiceBusTopicAuthorizationRule_requiresImport
=== PAUSE TestAccAzureRMServiceBusTopicAuthorizationRule_requiresImport
=== RUN   TestAccAzureRMServiceBusTopic_requiresImport
=== PAUSE TestAccAzureRMServiceBusTopic_requiresImport
=== CONT  TestAccAzureRMServiceFabricCluster_requiresImport
=== CONT  TestAccAzureRMServiceBusSubscriptionRule_requiresImport
=== CONT  TestAccAzureRMServiceBusTopic_requiresImport
=== CONT  TestAccAzureRMServiceBusTopicAuthorizationRule_requiresImport
=== CONT  TestAccAzureRMServiceBusSubscription_requiresImport
=== CONT  TestAccAzureRMServiceBusQueueAuthorizationRule_requiresImport
=== CONT  TestAccAzureRMServiceBusQueue_requiresImport
=== CONT  TestAccAzureRMServiceBusNamespace_requiresImport
=== CONT  TestAccAzureRMServiceBusNamespaceAuthorizationRule_requiresImport
--- PASS: TestAccAzureRMServiceFabricCluster_requiresImport (90.24s)
--- PASS: TestAccAzureRMServiceBusNamespace_requiresImport (423.63s)
--- PASS: TestAccAzureRMServiceBusQueue_requiresImport (431.73s)
--- PASS: TestAccAzureRMServiceBusTopic_requiresImport (433.10s)
--- PASS: TestAccAzureRMServiceBusSubscription_requiresImport (442.00s)
--- PASS: TestAccAzureRMServiceBusSubscriptionRule_requiresImport (448.58s)
--- PASS: TestAccAzureRMServiceBusQueueAuthorizationRule_requiresImport (843.96s)
--- PASS: TestAccAzureRMServiceBusTopicAuthorizationRule_requiresImport (844.02s)
--- PASS: TestAccAzureRMServiceBusNamespaceAuthorizationRule_requiresImport (890.23s)
PASS
ok  	github.com/terraform-providers/terraform-provider-azurerm/azurerm	982.277s
[11:46:55] kt@snowbook:~/hashi/..3../terraform-providers/terraform-provider-azurerm▸refactor/require_import/S-SQL$
```

**Comment by katbyte** [comment]
```
[19:16:23] kt@snowbook:~/hashi/..3../terraform-providers/terraform-provider-azurerm▸refactor/require_import/S-SQL$ testazure TestAccAzureRMS[in].*_requiresImport
==> Fixing source code with gofmt...
# This logic should match the search logic in scripts/gofmtcheck.sh
gofmt -s -w `find . -name '*.go' | grep -v vendor`
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./azurerm -v -test.run=TestAccAzureRMS[in].*_requiresImport -timeout 180m -ldflags="-X=github.com/terraform-providers/terraform-provider-azurerm/version.ProviderVersion=acc"
=== RUN   TestAccAzureRMSignalRService_requiresImport
=== PAUSE TestAccAzureRMSignalRService_requiresImport
=== RUN   TestAccAzureRMSnapshot_requiresImport
=== PAUSE TestAccAzureRMSnapshot_requiresImport
=== CONT  TestAccAzureRMSignalRService_requiresImport
=== CONT  TestAccAzureRMSnapshot_requiresImport
--- PASS: TestAccAzureRMSnapshot_requiresImport (205.82s)
--- PASS: TestAccAzureRMSignalRService_requiresImport (287.31s)
PASS
ok  	github.com/terraform-providers/terraform-provider-azurerm/azurerm	287.783s
```

**Comment by katbyte** [comment]
```
[19:22:06] kt@snowbook:~/hashi/..3../terraform-providers/terraform-provider-azurerm▸refactor/require_import/S-SQL$ testazure TestAccAzureRMSql.*_requiresImport
==> Fixing source code with gofmt...
# This logic should match the search logic in scripts/gofmtcheck.sh
gofmt -s -w `find . -name '*.go' | grep -v vendor`
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./azurerm -v -test.run=TestAccAzureRMSql.*_requiresImport -timeout 180m -ldflags="-X=github.com/terraform-providers/terraform-provider-azurerm/version.ProviderVersion=acc"
=== RUN   TestAccAzureRMSqlAdministrator_requiresImport
=== PAUSE TestAccAzureRMSqlAdministrator_requiresImport
=== RUN   TestAccAzureRMSqlDatabase_requiresImport
=== PAUSE TestAccAzureRMSqlDatabase_requiresImport
=== RUN   TestAccAzureRMSqlElasticPool_requiresImport
=== PAUSE TestAccAzureRMSqlElasticPool_requiresImport
=== RUN   TestAccAzureRMSqlFirewallRule_requiresImport
=== PAUSE TestAccAzureRMSqlFirewallRule_requiresImport
=== RUN   TestAccAzureRMSqlServer_requiresImport
=== PAUSE TestAccAzureRMSqlServer_requiresImport
=== RUN   TestAccAzureRMSqlVirtualNetworkRule_requiresImport
=== PAUSE TestAccAzureRMSqlVirtualNetworkRule_requiresImport
=== CONT  TestAccAzureRMSqlAdministrator_requiresImport
=== CONT  TestAccAzureRMSqlServer_requiresImport
=== CONT  TestAccAzureRMSqlVirtualNetworkRule_requiresImport
=== CONT  TestAccAzureRMSqlDatabase_requiresImport
=== CONT  TestAccAzureRMSqlElasticPool_requiresImport
=== CONT  TestAccAzureRMSqlFirewallRule_requiresImport
--- PASS: TestAccAzureRMSqlServer_requiresImport (143.20s)
--- PASS: TestAccAzureRMSqlFirewallRule_requiresImport (146.70s)
--- PASS: TestAccAzureRMSqlElasticPool_requiresImport (178.62s)
--- PASS: TestAccAzureRMSqlAdministrator_requiresImport (192.81s)
--- PASS: TestAccAzureRMSqlDatabase_requiresImport (284.27s)
--- PASS: TestAccAzureRMSqlVirtualNetworkRule_requiresImport (416.27s)
PASS
ok  	github.com/terraform-providers/terraform-provider-azurerm/azurerm	416.823s
[11:37:31] kt@snowbook:~/hashi/..3../terraform-providers/terraform-provider-azurerm▸refactor/require_import/S-SQL$
```

File: azurerm/resource_arm_servicebus_queue_authorization_rule_test.go [hunk 1]
```diff
@@ -100,6 +100,34 @@ func TestAccAzureRMServiceBusQueueAuthorizationRule_rightsUpdate(t *testing.T) {
...
+				Config: testAccAzureRMServiceBusQueueAuthorizationRule_base(tf.AccRandTimeInt(), testLocation(), true, false, false),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMServiceBusQueueAuthorizationRuleExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "listen", "true"),
+					resource.TestCheckResourceAttr(resourceName, "send", "false"),
+					resource.TestCheckResourceAttr(resourceName, "manage", "false"),
+				),
+			},
+			{
+				Config:      testAccAzureRMServiceBusQueueAuthorizationRule_requiresImport(tf.AccRandTimeInt(), testLocation(), true, false, false),
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_servicebus_queue_authorization_rule_test.go]: I believe we need to share the same unique ID here?

File: azurerm/resource_arm_servicebus_topic_authorization_rule_test.go [hunk 2]
```diff
@@ -59,6 +59,35 @@ func testAccAzureRMServiceBusTopicAuthorizationRule(t *testing.T, listen, send,
...
+				Config: testAccAzureRMServiceBusTopicAuthorizationRule_base(tf.AccRandTimeInt(), testLocation(), true, false, false),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMServiceBusTopicAuthorizationRuleExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "listen", "true"),
+					resource.TestCheckResourceAttr(resourceName, "send", "false"),
+					resource.TestCheckResourceAttr(resourceName, "manage", "false"),
+				),
+			},
+			{
+				Config:      testAccAzureRMServiceBusTopicAuthorizationRule_requiresImport(tf.AccRandTimeInt(), testLocation(), true, false, false),
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_servicebus_topic_authorization_rule_test.go]: this needs to share the same ID here

**Pull request merged**